DXRModules/DeusEx/Classes/DXRBingoCampaign.uc

```diff
@@ -192,7 +192,6 @@ function NewBingoBoard()
 
     dxr.flags.settings.starting_map = dxr.dxInfo.missionNumber * 10;
```

Comment:
I wonder if changing the starting_map can have any bad side effects here, probably would be best to store the old value, change it, generate the board, and then put it back to the old value

Reply:
Good idea.